src/AI/agents/shared/utils/langfuse_utils.py

```
@@ -243,6 +243,20 @@ def __exit__(self, *args):
 
+def delete_score(score_id: str) -> None:
+    """Remove a score by id. Used to clear user feedback on a trace."""
+    client = get_langfuse_client()
+    if not config.LANGFUSE_ENABLED:
+        return
+    if not client or not score_id:
+        return
+    try:
+        client.api.legacy.score_v1.delete(score_id)
+        log.debug("Langfuse score %s deleted", score_id)
+    except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)